docs/pages/guides/using-sentry.mdx

```diff
@@ -162,17 +162,18 @@ Add `expo.hooks` property to your project's **app.json** or **app.config.js** fi
 }
 ```
 
-The `authToken` value can be generated from the [Sentry API page](https://sentry.io/settings/account/api/).
+To upload the source map to Sentry, you need to create a [Sentry auth token](https://docs.sentry.io/product/cli/configuration/). After creating your own Sentry auth token [here](https://sentry.io/settings/account/api/), you can configure this token through the `SENTRY_AUTH_TOKEN` environment variable in [EAS Build](../build-reference/variables.mdx#using-secrets-in-environment-variables).
 
-<Collapsible summary="Prefer to use environment variables instead of storing values in app.json?">
+<Callout>
 
-You can also use environment variables for your config, if you prefer:
+Never commit secrets to your repository. Keep the Sentry auth token in a safe place, outside your repository and use the environment variable instead.
+
+</Callout>
```

Comment:
Instead of importing a component you can just use quote (`>`) Markdown syntax, which is translated to Callout.

Reply:
I tried, but I couldn't get the callout to be a warning callout :C